ai/STA/Tactic/demo_follow_robot.py

```diff
@@ -23,20 +22,20 @@ def __init__(self, game_state: GameState, player: Player, p_target: Pose=Pose(),
 
     def move_to_ball(self):
         self.status_flag = Flags.WIP
-        self.target = self.game_state.get_player_pose(self.robot_to_follow_id)
+        self.target = self.game_state.get_player(self.robot_to_follow_id).pose
```

Comment:
you can access .position directly to avoid indirection.

Reply:
But we don't need the position, we need the pose